static/js/units-i18n.js

```diff
@@ -57,12 +49,7 @@ export function getDistanceUnit(locale){
  * @returns {Object}
  */
 function getUnitsForLocale(locale) {
-  const language = locale.substring(0,2);
-  // Note: Getting region this way will be invalid if script tags are used in the future.
-  // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
-  // Optionally, we can use Intl.locale() but we would need to polyfill it for IE11
-  const region = locale.substring(3,5); 
-  const unitSystemFallback = METRIC;
```

Comment:
do we still need unitSystemFallback?

Reply:
The code is still using it. I just moved it below LOCALE_UNIT_MAP